internal/functions/coviddata/download-and-count-vaccinations.go

```diff
@@ -13,10 +13,34 @@ import (
 	"time"
 )
 
-func fetchVaccinationsData(client HTTPClient) (*VaccinationsAggregatedData, error) {
+// DownloadAndCountVaccinations downloads vaccinations metrics json and writes it to firestore
+func DownloadAndCountVaccinations(w http.ResponseWriter, r *http.Request) {
+	var ctx = context.Background()
+	logger := logging.FromContext(ctx).Named("DownloadAndCountVaccination")
+
+	spaceClient := http.Client{
```

Comment:
WTH is "space"? :smile:

Reply:
This naming is used in `DownloadCovidDataTotal` ..... 🤷

Reply:
OK, maybe fix it there too :joy: Wasn't it you who wrote it there?